src/pgduckdb_ddl.cpp

```diff
@@ -181,6 +187,7 @@ DuckdbHandleDDL(PlannedStmt *pstmt, const char *query_string, ParamListInfo para
 		char *access_method = stmt->accessMethod ? stmt->accessMethod : default_table_access_method;
 		bool is_duckdb_table = strcmp(access_method, "duckdb") == 0;
 		if (is_duckdb_table) {
+			pgduckdb::top_level_duckdb_ddl_type = pgduckdb::DDLType::CREATE_TABLE;
```

Comment:
Shouldn't we assert that this is `DDLType::NONE` before updating it? (same everywhere we update it). Or we're confident this can never happen?

Reply:
I'm pretty confident it cannot happen, but I guess it's better safe than sorry. So now it throws an error in that case.

Reply:
Should we do the same below (L216?)